manifests/s/Skiff/Skiff/1.0.9/Skiff.Skiff.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Skiff.Skiff
+PackageVersion: 1.0.9
+InstallerLocale: en-US
+InstallerType: msi
+ProductCode: '{47CE8D9A-107C-4F0D-B65C-990D299EA0BD}'
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
+Installers:
+- InstallerUrl: https://github.com/skiff-org/skiff-windows-app/releases/download/1.0.0.9/Skiff.Installer.msi
+  Architecture: x86

```

```suggestion
  Architecture: x64
```
Should this be a 64-bit application? Our `x86` validations are performed on 32-bit OSes.